src/Domain/Item/ContentType.ts

```diff
@@ -1,4 +1,5 @@
 export enum ContentType {
   Note = 'Note',
-  ItemsKey = 'SN|ItemsKey'
+  ItemsKey = 'SN|ItemsKey',
+  Extension = 'SF|Extension',
```

Comment:
```suggestion
  ServerExtension = 'SF|Extension',
```

This is what I would have called it earlier on if I understood what it was to become. ServerExtensions will likely be deprecated after we launch UserSettings, which will be used for cloud backup settings instead.